pmm/pmm2-ui-tests.groovy

```diff
@@ -132,51 +159,63 @@ pipeline {
             }
         }
         stage('Checkout Commit') {
-             when {
+            when {
                 expression { env.GIT_COMMIT_HASH.length()>0 }
             }
             steps {
                 sh 'git checkout ' + env.GIT_COMMIT_HASH
             }
         }
-        stage('Start staging') {
-            steps {
-                runStaging(DOCKER_VERSION, CLIENT_VERSION, '--addclient=ps,1 --addclient=mo,2 --with-replica --addclient=pgsql,1 --addclient=pxc,1 --with-proxysql --pmm2 --setup-alertmanager --add-annotation', CLIENT_INSTANCE, SERVER_IP)
+        stage('Start PMM Server') {
```

Comment:
Better to be `Start PMM Server Instance` as on line 177, there's: `Start PMM Client Instance`

Reply:
sure will make that correction thanks.